include/eve/module/math/regular/deginrad.hpp

```diff
@@ -47,8 +58,21 @@ namespace eve
 //!  @godbolt{doc/math/regular/deginrad.cpp}
 //!  @}
 //================================================================================================
-
-EVE_MAKE_CALLABLE(deginrad_, deginrad);
+inline constexpr auto deginrad = functor<deginrad_t>;
 }
 
-#include <eve/module/math/regular/impl/deginrad.hpp>
+namespace eve::detail
+{
+  template<floating_ordered_value T, callable_options O>
+  EVE_FORCEINLINE constexpr T
+  deginrad_(EVE_REQUIRES(cpu_), O const &, T const& a) noexcept
+  {
+    if constexpr( has_native_abi_v<T> )
+    {
+      auto ridh = ieee_constant<0x1.1de0000p-6f, 0x1.1df46a0000000p-6>(eve::as<T>{});
+      auto ridl = ieee_constant<0x1.46a2520p-18f, 0x1.294e9c8ae0ec6p-33>(eve::as<T>{});
+      return fma(a, ridl, a * ridh);
+    }
+    else return apply_over(deginrad, a);
```

Comment:
@jfalcou - doesn't element wise do this?
Even if it doesn't. I would expect the `if constexpr` to be unnecessary. Is there a reason why it's necessary?

Reply:
It is  indeed in flux. We found elementwise_callable needs a slight more massaging to get there.
This is noted and will be done after this PR so we can get rid of apply_over everywhere.